src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS_CIRCLE = 10;
+    private static final int MAX_RANDOM = 100;
+    private static final int NUMBER_FIGURE = 5;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomIsoscelesTrapezoid() {

```

Comment:
Private ones (methods, classes) should be after public ones.

```suggestion
    private Figure getRandomIsoscelesTrapezoid() {

```